public/javascripts/SVLabel/src/SVLabel/label/Label.js

```diff
@@ -423,6 +423,16 @@ function Label (svl, pathIn, params) {
             // Renders the label image.
             path.render2(ctx, pov);
 
+            ctx.lineWidth = .7;
```

Comment:
Can you add a comment above this block of code so that we know that this code draws the outline?